### DISCUSSION TITLE
can anyone help me why, I just uploaded Adonis to cpanel using node js setup and using mysql database

### DISCUSSION THREAD

**Discussion by nabils24** [adonisjs/core#4170] [Help]
can anyone help me why, I just uploaded Adonis to cpanel using the node js setup and using the mysql database, this error occurs when I want to send a request with the POST method from postman to create a new user

![image](https://github.com/adonisjs/core/assets/107533428/4ee0afa3-764f-426d-a048-8a09802a1bdd)

**Comment by McSneaky** [comment]
Have you installed `mysql2`? Or is it in list of your packages?

**Comment by prabhaw** [comment]
How do you deploy in cPanel? Could you please explain? I'm trying to do so
